src/NuGet.Indexing/FileLoader.cs

```diff
@@ -28,6 +28,14 @@ public JsonReader GetReader(string name)
             return new JsonTextReader(new StreamReader(fullName));
         }
 
+        public DateTime? GetLastUpdateTime(string name)
```

Comment:
we have a lot of inconsistencies in the usage of our time. I am trying to return all the times in UTC, however changing this to DateTimeOffset would mean changing other data(reopen time, reload time) in the diag endpoint to be consistent. I would rather keep this the same DateTime object, just ensuring that it is UTC, we need to update our commit-time-stamp values to be consistent as well to be UTC. I am not sure if this data is being used anywhere else and being parsed correctly or not, so I'd be hesitant to change it here.